regparser/tree/struct.py

```diff
@@ -51,6 +52,10 @@ def depth(self):
         else:
             return len(self.label)
 
+    @staticmethod
+    def is_markerless_label(label):
+        return re.match(r'p\d+', label[-1])
```

Comment:
It'd probably be a good idea to compile this regex ahead of time. Maybe add a class-level property?
```python
MARKERLESS_REGEX = re.compile(r'p\d+')
```

Unrelated: It would be nice if this didn't explode when label was empty.

Also unrelated: At points, we actually use `regparser.tree.depth.markers.MARKERLESS` to indicate nodes which haven't been assigned a unique number yet. Should those also satisfy this predicate?